### PR TITLE
dnuzum todo list

### PR DESCRIPTION
**Pull request by dnuzum**
Comfort: 4
Completeness: 5

Obviously, I had the benefit of following along with Brian to make this easier, but I still played around a bit and attempted to get a toggle working to apply and remove the strikethrough I used.

I wanted a strikethrough so not only could a person see everything they've done (and be proud), but undo it as well if they realized it's not complete without having to add it back into the list(uh-oh). 

I read in a few place toggle does and doesn't work here and there. I'm hoping to avoid writing another function for it. I thought the toggleClass would have worked. I've left it in a commented line. Help? :)

### CODE REVIEW

**Comment by dnuzum** [comment]
I fixed the pull request for toggle. Yay bugfixes!

**Comment by dnuzum** [comment]
Realized I broke strikethrough toggle a bit. Fixed it and added delete.

**Comment by dnuzum** [comment]
Added instructions because I realized the strikethrough wasn't obviously a function that existed.

**Comment by dnuzum** [comment]
Really wanted sweetAlert to work, but couldn't integrate the examples into my project. Got sortable to work though!

File: index.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  <form id="to-do">
+    <input type="text" id="newTodo" placeholder="Enter to-do">
+    <button type="submit">Submit to list!</button>
+  </form>
+  <p>Click item to strikethrough. Click X to delete.<br/>Click and Drag to sort.</p>
+
+</body>
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.0/jquery.min.js"></script>
+<script src="https://code.jquery.com/ui/1.11.4/jquery-ui.js"></script>
+<script src ="js/sweetalert.min.js"></script>
```
Comment (jsakalys) [inline index.html:26]: Watch out for spaces here, it's generally a bad idea.

```
<script src="js/sweetalert.min.js"></script>
<script src="js/script.js"></script>
```

File: js/script.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  //   $(this).parent().addClass("strike");
+  // });
+	  $('ul').css('cursor', 'pointer');
+	  $('ul').on('click', 'li', function() {
+	  // console.log('li clicked!');
+	  $(this).toggleClass("strike");
+	});
+	  $('ul').on('click', 'li span', function() {
+	  // console.log('span clicked!');
+	  $(this).parent().remove();
```
Comment (jsakalys) [inline js/script.js:29]: Sweet alert worked for me when I called it upon removing an item:

```
swal('Removed!');
```

**Comment by jsakalys** [comment]
Everything looks and works great. Nicely done!

**Pull request closed without merging**